include/cartesian_geom/point.h

```diff
@@ -0,0 +1,105 @@
+#ifndef POINT_H
+#define POINT_H
+
+
+template <class Cr>
+class point
+{
+public:
+    typedef typename Cr::RT 	K;
+    int d;
+    typedef std::vector<K> coeff;
+    coeff coeffs;
```

Comment:
types can start with capital to not be confused with objects

Reply:
Done